docs/guides/mobile-plugin-gma-android.md

```diff
@@ -29,9 +29,13 @@ This plugin is in the following open-source GitHub repository:
 
 To run this plugin, install the following:
 
-- [Google Mobile Ads SDK v22.0.0+](https://developers.google.com/admob/android/rel-notes)
-- [UID2 SDK for Android v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk) (for documentation, see [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
-- [UID2 Android GMA Plugin v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk-gma/)
+1. Google Mobile Ads SDK v22.0.0 or later:
+   - [SDK](https://developers.google.com/admob/android/sdk)
+   - [Release notes](https://developers.google.com/admob/android/rel-notes)
+1. UID2 SDK for Android v0.5.0:
+   - [SDK](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk)
+   - [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
```

Comment:
Reference Guide link is not needed.

Reply:
@bleege  I'd like to include a link to the SDK doc in the plug-in doc... it isn't mentioned elsewhere in the doc. Is it a problem? LMK if so and I'll take it out.
(fixed the syntax error there).

Reply:
That makes sense to me now.  Let's leave it in.  👍